core/src/main/resources/spline.default.properties

```diff
@@ -36,6 +36,10 @@ spline.lineageDispatcher.http.className=za.co.absa.spline.harvester.dispatcher.H
 spline.lineageDispatcher.http.producer.url=
 spline.lineageDispatcher.http.timeout.connection=2000
 spline.lineageDispatcher.http.timeout.read=120000
+# disables protocol version negotiation and sets the version to 1.2 ( LATEST could be used as well)
+# spline.lineageDispatcher.http.apiVersion=1.2
+# disables requestCompression negotiation and sets it to false
+# spline.lineageDispatcher.http.requestCompression=false
```

Comment:
```suggestion
# Should the payload be compressed.
# If unspecified, the request compression is negotiated with the endpoint, otherwise it's disabled.
# spline.lineageDispatcher.http.requestCompression=true | false
```